lightly/active_learning/config/sampler_config.py

```diff
@@ -11,22 +11,22 @@ class SamplerConfig:
     Attributes:
         method:
             The method to use for sampling, e.g. RANDOM.
-        batch_size:
-            The maximum number of samples to be chosen by the sampler.
+        n_samples:
```

Comment:
Why did you switch from `batch_size` to `n_samples`?

Reply:
Before doing that, I had the following wokflow:
- I want to have n_samples and M of them are in the preselected set.
- First I have to calculate M and substract it to calculate batch_size = n_samples-M
- Inside the sampling itself, the M has to be added again, thus the preselected tag is loaded from the API to calculate M and then again n_samples = batch_size + M is set.

Instead of substracting M first, only to add it afterwards and peforming API request for each of this, it is much easier to have the sampler_config to store the n_samples instead of the batch_size.

Reply:
But is it not more user-friendly if we use the batch size?

Reply:
A user could say:
- I am planning my AL run to have 5%, then 10%, then 20% of the data labeled -> n_samples is more user friendly
- I found that my model is not good enough, I want to sample batch_size additional samples -> batch_size is more user friendly.

As the ActiveLearningAgent.query() returns the full labeled set with len(labeled_set)==n_samples, having the config with n_samples fits better to the overall workflow.